src/util/util.h

```diff
@@ -109,7 +109,6 @@ extern int socket_activated;
 #define FLAGS_DAEMON 0x0001
 #define FLAGS_INTERACTIVE 0x0002
 #define FLAGS_PID_FILE 0x0004
-#define FLAGS_GEN_CONF 0x0008
 #define FLAGS_NO_WATCHDOG 0x0010
```

Comment:
Maybe a comment saying that flag `0x0008` aka `1<<3` was removed and can be reused in future would be a good reminder. Or if you think that it should not be reused in future, a comment would be good as well.

Reply:
Done.

Reply:
Thanks